src/tools/python.jam

```diff
@@ -545,6 +545,13 @@ local rule compute-default-paths ( target-os : version ? : prefix ? :
     else
     {
         includes ?= $(prefix)/include/python$(version) ;
+        if $(sys.abiflags)
+        {
+            # Calling "sys.abiflags" does not throw an exception, which 
+            # indicates this is neither a Python 2 build, nor a build from
+            # source distro.
+            includes ?= $(includes)$(sys.abiflags) ;
```

Comment:
I think this should be `=` rather than `?=` otherwise it won't change anything.

Reply:
How do `=` and `?=` differ? I'm not very familiar with boost.build and I thought `?=` would skip assignment if variable is already set and `=` would force an assignment.

Reply:
I'm not that familiar either but when I was trying this patch locally it didn't solve the problem as-is. The setting of `includes` a few lines above is unconditional so it seems this line with a `?=` is resulting in a no-op.